packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -48,7 +45,7 @@ Users can drag-and-drop appointments between groups or use the appointment edit
 
 Timetable cells have the `groupingInfo` property that lists groups to which a particular cell belongs. Use this property to customize cells depending on their group.
 
-In addition, the `GroupingPanel` plugin has properties that customize the grouping panel: `horizontalLayoutComponent`, `rowComponent`, and `cellComponent`.
+In addition, the `GroupingPanel` plugin has properties that customize the group panel: `horizontalLayoutComponent`, `rowComponent`, and `cellComponent`.
```

Comment:
Can we improve on this by saying:
In addition, you can use the following `GroupingPanel` plugin properties to customize a group panel: `horizontalLayoutComponent`, `rowComponent`, and `cellComponent`.